sys/net/network_layer/sixlowpan/lowpan.c

```diff
@@ -1418,7 +1419,10 @@ void lowpan_iphc_decoding(uint8_t *data, uint8_t length, net_if_eui64_t *s_addr,
             case (0x02): {
                 /* 16-bits */
                 memcpy(&(ipv6_buf->destaddr.uint8[0]), &ll_prefix[0], 2);
-                memset(&(ipv6_buf->destaddr.uint8[2]), 0, 12);
+                ipv6_buf->destaddr.uint16[1] = HTONS(0x0000u);
```

Comment:
we don't need to do HTONS on a null value, but I decided to put it there anyway in case someone later decided to change the number to a non-palindrome integer